app/templates/service_update_audits.html

```diff
@@ -139,26 +139,19 @@ <h3 class="validation-masthead-heading" id="validation-masthead-heading">
               {% endcall %}
             {% endcall %}
 
-            <!--
-            <nav role="navigation">
-                <ul class="previous-next-navigation">
-
-                    <li class="previous">
-                        <a href="">
-                            Previous <span class="visuallyhidden">page</span>
-                            <span class="page-numbers">1 of 3</span>
-                        </a>
-                    </li>
-
-                    <li class="next">
-                        <a href="">
-                            Next <span class="visuallyhidden">page</span>
-                            <span class="page-numbers">2 of 3</span>
```

Comment:
Do we mind not having the "Page 2 of 3" etc. any more?  Or did this never work in the first place anyway?